@xen-orchestra/openflow/README.md

```diff
@@ -0,0 +1,158 @@
+# @xen-orchestra/log [![Build Status](https://travis-ci.org/vatesfr/xen-orchestra.png?branch=master)](https://travis-ci.org/vatesfr/xen-orchestra)
+
+> Pack and unpack OpenFlow messages
+
+## Install
+
+Installation of the [npm package](https://npmjs.org/package/@xen-orchestra/openflow):
+
+```
+> npm install --save @xen-orchestra/openflow
+```
+
+## Usage
+
+Unpacking a received OpenFlow message from a socket:
+
+```js
+import openflow from '@xen-orchestra/openflow'
+import Stream from '@xen-orchestra/openflow/dist/stream'
```

Comment:
Either re-export this from the main module or add an entry point at root level to avoid exposing dev detail (`dist` folder).

`stream.js`:
```js
module.exports = require('./dist/stream')
```